src/index.js

```diff
@@ -47,30 +48,34 @@ export default function supernova(env) {
       const [opts] = useState(options);
       const selectionsAPI = useSelections();
       const constraints = useConstraints();
-      const [selections] = useState({ api: selectionsAPI, setState: setSelectionState, linked: false });
+      const [selectionsAndZoom] = useState({ api: selectionsAPI, setState: setSelectionState, linked: false, transform: {} });
```

Comment:
If we store both zoom and translation, it should be `selectionsAndTransform` I guess